From ba181b3088d7783b623b843da3e6518d9f1ed9fc Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 28 Dec 2022 01:01:40 +0800
Subject: [PATCH 22/28] Revert "drm/vc4: Fix missing
 platform_unregister_drivers() call in vc4_drm_register()"

This reverts commit f3aa8a7d9550a5f9ccbd4e0ed914a065d0741cc8.
---
 drivers/gpu/drm/vc4/vc4_drv.c | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/drivers/gpu/drm/vc4/vc4_drv.c b/drivers/gpu/drm/vc4/vc4_drv.c
index 888aec1bbeee6..52426bc8edb8b 100644
--- a/drivers/gpu/drm/vc4/vc4_drv.c
+++ b/drivers/gpu/drm/vc4/vc4_drv.c
@@ -404,12 +404,7 @@ static int __init vc4_drm_register(void)
 	if (ret)
 		return ret;
 
-	ret = platform_driver_register(&vc4_platform_driver);
-	if (ret)
-		platform_unregister_drivers(component_drivers,
-					    ARRAY_SIZE(component_drivers));
-
-	return ret;
+	return platform_driver_register(&vc4_platform_driver);
 }
 
 static void __exit vc4_drm_unregister(void)
-- 
2.25.1

